src/main/java/org/icgc/argo/program_service/controller/PublicProgramController.java

```diff
@@ -29,4 +30,10 @@ public ResponseEntity<ProgramDTO> getPublicProgramData(
 
     return ResponseEntity.ok(programDTO);
   }
+
+  @GetMapping(value = "/registeredPrograms")
```

Comment:
Can we simplify this to just `/programs`? I think the registered part is implied everywhere except while querying the db.

Reply:
Sure @joneubank. I will make the change